pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/supported-databases/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: layout.pug
+navigationTitle: Supported Databases
+excerpt: Supported Databases
+title: Getting Started
+menuWeight: 25
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+
+## Supported Databases
+
+The DC/OS {{ model.techName }} supports the following relational databases (RDBMS):
+ - Derby
+ - MySQL
+ - Percona-MySQL
+
+### Install the {{ model.techShortName }} with Derby
+By default, {{ model.techName }} configuration is set to the default embedded Derby database for development usage. However, it is not intended for use beyond simple testing.
+
+```
+dcos package install {{ model.packageName }}
+```
+
+
+### Install the {{ model.techShortName }}  with MySQL
+The {{ model.techShortName }} can also be backed by a MySQL database. This is recommended for production use.
```

Comment:
We should recommend percona-mysql for production use, rather than Vanilla MySQL as it is more robust and provides HA functionality.

Reply:
@vibhujain , `Hive-Metastore` needs to be compatible with all 4 strict modes of `percona-mysql`. Currently it supports only 2 strict modes.

Reply:
Maybe we should remove that comment and just say that using Derby is not recommended for production use.